templates/index.html

```diff
@@ -23,7 +23,9 @@ <h1>QuickNotes</h1>
                             id="contents"></textarea>
       </div>
 
-      <button id="submit">Save</button> <button id="clear">Delete</button>
+      <button id="submit" onclick="javascript:buttonAction();">Save</button> <button id="clear">Delete</button>
+
+         <!-- <button type="button" id="test" onclick="javascript:test.onclick();">Test</button> -->
```

Comment:
Same as the JS stuff, don't leave this in here.

Reply:
I deleted the comments completely. Testing works fine without a button needed